terraform/aws/apps-deployment/main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     region = "us-east-1"
   }
   required_providers {
-    aws = "~> 3.17.0"
+    aws = "~> 3.37.0"
```

Comment:
is this for the runtime in lambda?

Reply:
Yeap. The old provider does not support newer nodejs versions. 

FYI  Error: 
```
expected runtime to be one of [nodejs nodejs4.3 nodejs6.10 nodejs8.10 nodejs10.x nodejs12.x java8 java8.al2 java11 python2.7 python3.6 python3.7 python3.8 dotnetcore1.0 dotnetcore2.0 dotnetcore2.1 dotnetcore3.1 nodejs4.3-edge go1.x ruby2.5 ruby2.7 provided provided.al2], got nodejs14.x"
```